app/scenes/GroupMembers/GroupMembers.js

```diff
@@ -20,112 +14,101 @@ import PaginatedList from "components/PaginatedList";
 import Subheading from "components/Subheading";
 import AddPeopleToGroup from "./AddPeopleToGroup";
 import GroupMemberListItem from "./components/GroupMemberListItem";
+import useCurrentUser from "hooks/useCurrentUser";
+import useStores from "hooks/useStores";
 
 type Props = {
-  ui: UiStore,
-  auth: AuthStore,
   group: Group,
-  users: UsersStore,
-  policies: PoliciesStore,
-  groupMemberships: GroupMembershipsStore,
-  t: TFunction,
 };
 
-@observer
-class GroupMembers extends React.Component<Props> {
-  @observable addModalOpen: boolean = false;
+function GroupMembers({ group }: Props) {
+  const [addModalOpen, setAddModalOpen] = React.useState();
+  const { users, groupMemberships, policies, ui } = useStores();
+  const { t } = useTranslation();
+  const can = policies.abilities(group.id);
 
-  handleAddModalOpen = () => {
-    this.addModalOpen = true;
-  };
+  // validate user is not null
+  useCurrentUser();
```

Comment:
```suggestion
```

This line alone doesn't do anything unless you use the return value?

Reply:
There was this guard condition `if (!user) return null;`. It was the only use of `user` in the code. I thought there must be some bug or validation requirement, thus invoking this hook to ensure that the user is present.

Reply:
I think you're safe to remove as it's unused, must have been a copy/paste error